### ISSUE TITLE
JSON exporter does not save all exported records

### ISSUE THREAD

**Issue by damian0604** [uvacw/inca#368]
returns file with 6 lines even though 5515 results 

```
In [4]: e = 'plantaardig*'

In [5]:  myinca.importers_exporters.export_json_file(query="text:{}".format(e), destination = '/mnt/elastic/voordamian/onderwijs/{}'.format(e))
INFO:INCA.core.search_utils:String input: searching for text:plantaardig*
INFO:INCA.core.search_utils:returning 0 of 5515
...
```
but:
```
damian@packer-ubuntu-16:~/inca-temp/inca$ wc -l /mnt/elastic/voordamian/eten/plantaardig\*.json 
6 /mnt/elastic/voordamian/eten/plantaardig*.json
```

**Comment by damian0604** [comment]
probably:
https://github.com/uvacw/inca/blob/development/inca/core/import_export_classes.py
at the very end in the loop, the file is reopened for writing instead of appending

**Comment by damian0604** [comment]
fixed in PR #369
